src/clarity-angular/popover/dropdown/dropdown-item.ts

```diff
@@ -20,12 +20,12 @@ import {IfOpenService} from "../../utils/conditional/if-open.service";
 })
 export class DropdownItem {
 
-    constructor(private _dropdown: Dropdown, private el: ElementRef, private ifOpenService: IfOpenService) {
+    constructor(private dropdown: Dropdown, private el: ElementRef, private ifOpenService: IfOpenService) {
     }
 
     @HostListener("click")
     onDropdownItemClick(): void {
-        if (this._dropdown.isMenuClosable && !this.el.nativeElement.classList.contains("disabled")) {
+        if (this.dropdown.isMenuClosable && !this.el.nativeElement.classList.contains("disabled")) {
```

Comment:
I thought that was already our recommendation (buttons that can actually be disabled or switch between label and link), and that code was here only for people ignoring it. If it's not the case then yes, we should document this recommendation.